cloudify_gcp/compute/instance.py

```diff
@@ -379,11 +380,26 @@ def create(instance_type,
             zone = props['zone'] = utils.get_gcp_resource_name(
                     gcp_config['zone'])
 
+    disks = [
+            disk.target.instance.runtime_properties[constants.DISK]
+            for disk
+            in utils.get_relationships(
+                ctx,
+                filter_resource_types='compute#disk'
+                )
+            ]
+    disks.sort(key=lambda disk: disk['boot'], reverse=True)
```

Comment:
Wouldn't it be better to filter only boot disks and then ask if there's more than 1 ?